egs2/voxceleb/spk1/conf/train_RawNet3.yaml

```diff
@@ -0,0 +1,30 @@
+# RawNet3 reproduce recipe configuration.
+encoder: rawnet3
+#encoder_conf:  # uncomment for customized arguments
+
+#model_conf:    # uncomment for customized arguments
+
+frontend: raw
+
+projector: rawnet3
+projector_conf:
+  input_size: 3072
+  output_size: 256
+
+
+preprocessor: spk
+preprocessor_conf:
+  utt2spk: dump/raw/voxceleb2_dev/utt2spk
+  spk2utt: dump/raw/voxceleb2_dev/spk2utt
```

Comment:
Do we need this part (or can we deal with it in the template if possible?)

Reply:
@ftshijt , thank you for the valuable comments. 
It would be great to pass this arguments in the template. 

It's a bit confusing to me at the moment, between the arguments which to be given by bash and the ones to be given by yaml. 
Do you think you can help me with one commit which let me freely access `utt2spk` inside the python script, which will be under `${train_set}/utt2spk` in the bash script? 

If I get this, I can reflect your next comment simply by 

```
set_cls = set()
for utt in utt2spk:
    set_cls.add(utt.strip().split(" ")[1])
nClasses = len(set_cls)
```

Reply:
Using the path name specific to the recipe would not be very robust.
Is it possible to change it?